wallet-api/src/Ledger/Validation.hs

```diff
@@ -180,11 +181,17 @@ newtype ValidatorHash =
     deriving stock (Eq, Generic)
     deriving newtype (Serialise)
 
+validatorHashToText :: ValidatorHash -> Text
+validatorHashToText = TE.decodeUtf8 . Base64.encode . BSL.toStrict . serialise
```

Comment:
@j-mueller should this be `Base16`?

Reply:
yes! Good catch!

Reply:
there are a few places where we use Base64, I suggest we have a separate PR for this. @j-mueller is there anything preventing you approving this now?

Reply:
The separate PR already exists (#717).  But yeah let's merge this and I'll rebase 717 onto it.